### PR TITLE
Configurable iCal Export and Preview

### PR DESCRIPTION
**Pull request by Clashsoft**
image

### CODE REVIEW

File: apps/frontend/src/app/poll/ical/ical-config.ts [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+  customTitle?: string;
+
+  @Presentation({
+    description: 'Include events without participants in the calendar.',
+  })
+  @IsBoolean()
+  emptyEvents = false;
+
+  @Presentation({
+    description: 'Only include booked events with participants in the calendar.',
```
Comment (Morphclue) [inline apps/frontend/src/app/poll/ical/ical-config.ts:19]:
```suggestion
    description: 'Include booked events with participants in the calendar.',
```

File: apps/frontend/src/app/poll/ical/ical.component.html [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+      <div class="card-body">
+        <table>
+          <tr>
+            <th>
+              <i class="bi-square-fill text-primary-emphasis"></i>
+            </th>
+            <td>
+              <h5 class="card-title">
+                {{ config.customTitle ?? poll?.title }}:
+                {{ exampleEvent?._participants?.[0]?.name }}
```
Comment (Morphclue) [inline apps/frontend/src/app/poll/ical/ical.component.html:23]:
```suggestion
```
Imo it's weird having the name of the first participant in the title.

File: apps/frontend/src/app/poll/ical/ical.component.html [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+    <div class="card">
+      <div class="card-body">
+        <table>
+          <tr>
+            <th>
+              <i class="bi-square-fill text-primary-emphasis"></i>
+            </th>
+            <td>
+              <h5 class="card-title">
+                {{ config.customTitle ?? poll?.title }}:
```
Comment (Morphclue) [inline apps/frontend/src/app/poll/ical/ical.component.html:22]:
```suggestion
                {{ config.customTitle ?? poll?.title }}
```
If I type a title and remove it afterwards no name for the customTitle is given.

**Review by Morphclue — CHANGES_REQUESTED** [review]

File: apps/frontend/src/app/poll/ical/ical.component.html (hunk 2 above)
Comment (Clashsoft) [reply apps/frontend/src/app/poll/ical/ical.component.html:23]: This is intentional. With a 1-1 poll you immediately see in your calendar who will show up in the next slot. It was very useful for the STC presentations.

**Pull request merged**